src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -155,6 +156,9 @@ export const routes: Routes = {
       />
     ),
   },
+  PROFILE_SETTINGS: {
```

Comment:
If you put this route as a subroute of Profile, and add `isNavStrict`, you will keep `My Profile` in the menu active.

Reply:
Yeah but I'm not sure we want to do that since there is no really link from these two pages

Reply:
`/my-profile` -> `/my-profile/settings`